src/qibolab/instruments/zhinst.py

```diff
@@ -1055,24 +1057,30 @@ def fast_reset(self, exp, qubits, fast_reset):
                         exp.play(signal=f"drive{q}", pulse=pulse.zhpulse)
 
     @staticmethod
-    def rearrange_sweepers(sweepers):
-        """Rearranges sweepers from qibocal based on device hardware limitations"""
-        rearranging_axes = [[], []]
+    def rearrange_sweepers(sweepers: List[Sweeper]) -> Tuple[np.ndarray, List[Sweeper]]:
+        """
+        Rearranges sweepers from qibocal based on device hardware limitations.
+
+        Frequency sweepers must be applied before bias or amplitude sweepers.
+
+        Args:
+            sweepers (list[Sweeper]): list of sweepers used in the experiment.
+
+        Returns:
+            rearranging_axes (np.ndarray): array of shape (2,) and dtype=int containing
+                the indexes of the sweepers to be swapped. Defaults to np.array([0, 0])
+                if no swap is needed.
+            sweepers (list[Sweeper]): updated list of sweepers used in the experiment. If
+                sweepers must be swapped, the list is updated accordingly.
+        """
+        rearranging_axes = np.zeros(2, dtype=int)
```

Comment:
I am assuming that this driver supports up to two sweepers only, right?

Reply:
No, more sweepers are supported, but in practice that function was already considering using only two sweepers.